src/common/src/config.rs

```diff
@@ -360,6 +372,18 @@ mod default {
     pub fn developer_enable_executor_row_count() -> bool {
         false
     }
+
+    pub fn developer_hash_agg_cache_size() -> usize {
```

Comment:
How about `unsafe_`?

Reply:
Sure, and I will refactor to the true SIZE first.